asv_bench/benchmarks/groupby.py

```diff
@@ -235,7 +235,7 @@ def time_different_python_functions_multicol(self, df):
         df.groupby(['key1', 'key2']).agg([sum, min, max])
 
     def time_different_python_functions_singlecol(self, df):
-        df.groupby('key1').agg([sum, min, max])
+        df.drop(['key2'], axis=1).groupby('key1').agg([sum, min, max])
```

Comment:
I think the problem here is that performance issues are going to be ambiguous between the drop and the groupby ops. Do you see a better way to refactor this test (maybe to a different class?) to get the intended effect?

Reply:
Yeah, I'd just change `setup_cache()` to `setup()`, but that'd require slightly changing all the tests in this suite.